src/Routing/Router.php

```diff
@@ -924,4 +928,29 @@ public static function setRouteCollection(RouteCollection $routeCollection): voi
     {
         static::$_collection = $routeCollection;
     }
+
+    /**
+     * Inject route defaults from `_path` key
+     *
+     * @param array $url Route array with `_path` key
+     * @return array
+     */
+    protected static function unwrapShortString(array $url)
+    {
+        foreach (['plugin', 'prefix', 'controller', 'action'] as $key) {
+            if (array_key_exists($key, $url)) {
+                throw new RuntimeException("The `_path` string cannot be overriden by `$key` key.");
+            }
+        }
+
+        $url += RouteBuilder::parseShortString($url['_path']);
+        $url += [
+            'plugin' => null,
+            'prefix' => false,
```

Comment:
Why are we using plugin and prefix differently? I would assume in 4.x both to be using the same "falsely" value, probably `null` as cleaner one.

Reply:
`plugin`: `null` is used everywhere.

`prefix`: `false` is used in docs, mix of `false`/`null` in test cases.

Reply:
`false` and `null` for `prefix` have different meanings. `null` will use current prefix if available, `false` will always unset/disable prefix.

Reply:
Given this, shouldn't prefix then also be null by default?

Reply:
It should stay `false` here.

One of purpose of this PR is to avoid specifying `prefix => false` in reusable elements/cells/emails/pdfs, so it removes plugin/prefix from current request.

I would like avoid workaround I sometimes use: in email, use links without `prefix => false` , but replace current request while sending email:
```php
$params = $controller->getRequest()->getAttribute('params');
unset($params['prefix']);
Router::pushRequest($controller->getRequest()->withAttribute('params', $params));

// send email

Router::popRequest();
```

Reply:
But why then not for plugin too?

Reply:
Plugin can be removed by both null and false.
I've never seen `plugin => false` anywhere and I didn't know `false` works too.

Reply:
If you want to generate a non plugin URL from within a plugin you would need to use `plugin => false`

Reply:
I've updated to `'plugin' => false` here